tests/chainer_tests/functions_tests/pooling_tests/test_roi_align_2d.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'sampling_ratio': [0, 1, 2],
```

Comment:
If you agree to my suggestion for allowing a tuple as `sampling_ratio`, please add some test case for it.